Docs for IAMMETER devices (#11630)
home-assistant/home-assistant.github.io: source/_integrations/iammeter.markdown

---
title: "iammeter"
description: "Instructions on how to integrate IAMMETER sensor within Home Assistant."
logo: iammeter-logo.png
ha_category:
  - Energy
  - Sensor
ha_release: 0.104
ha_iot_class: Local Polling
---

> [IAMMETER](https://www.iammeter.com) is cloud energy management platform. Our Bi-Directional Wi-Fi Energy Meters can measure the real-time reading and report to IAMMETER server. These meters can also be easily integrated into home assistant system. We have both single phase (WEM3080, WEM3162) and three phase (WEM3080T) meters.

Reviewer: ```suggestion
`iammeter` provides real-time reading of single phase (WEM3080, WEM3162) and three phase (WEM3080T) meters from [IAMMETER](https://www.iammeter.com) over Wi-Fi.
```

## Configuration

> Use the meter's sensors in your installation, and add the following to your configuration.yaml file:

Reviewer: ```suggestion
To use this sensor in your installation, add the following to your `configuration.yaml` file:
```
